packages/remix/src/utils/instrumentServer.ts

```diff
@@ -112,7 +113,7 @@ function makeWrappedDocumentRequestFunction(
     request: Request,
     responseStatusCode: number,
     responseHeaders: Headers,
-    context: Record<symbol, unknown>,
+    context: EntryContext,
```

Comment:
q: Why do we need this type change here?

Reply:
it complained about this not matching, I'll look again in detail.

Reply:
actually, the `HandleDocumentRequestFunction` has the context defined this way, and apparently in newer TS this is not satisifed by `Record<symbol, unknown>` anymore - which I guess makes sense, as the type is:

```ts
export interface EntryContext {
  [name: string]: any;
}
```

Reply:
Makes sense 👍